extensions/notebook/src/extension.ts

```diff
@@ -83,6 +83,14 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 		return dialog.createDialog();
 	}));
 
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.undoNotebookView', async () => {
```

Comment:
Notebook views means something else. Can we put "TreeView" instead of "View" here?